Test/Tests/RealtimeClientPresenceTests.swift

```diff
@@ -128,7 +128,7 @@ class RealtimeClientPresenceTests: XCTestCase {
         XCTAssertFalse(channel.internal.presence.syncComplete)
     }
 
-    func skipped__test__010__Presence__ProtocolMessage_bit_flag__when_members_are_present() throws {
+    func test__FLAKY__010__Presence__ProtocolMessage_bit_flag__when_members_are_present() throws {
```

Comment:
We should be renaming tests now right? `FLAKY` keyward can be removed if they are not flaky anymore wdyt

Reply:
I wanted to do it [later](https://github.com/ably/ably-cocoa/issues/1887) when absolutely sure they are alright.